src/test/kotlin/data_structure/LinkedListTest.kt

```diff
@@ -0,0 +1,32 @@
+package data_structure
```

Comment:
Package name should be plural -> `data_structures`

Reply:
the package name changed in the next commit